geomstats/geometry/hypersphere.py

```diff
@@ -301,6 +301,7 @@ def random_uniform(self, n_samples=1):
         while True:
             norms = gs.linalg.norm(samples, axis=1)
             indcs = gs.isclose(norms, 0.0)
+            indcs = gs.cast(indcs, gs.int32)
```

Comment:
Since `np.sum` works with booleans, could we modify `gs.sum`, in order to do the cast to `gs.int32` in the tf/torch's backend function and not in the main code?